ComfyUI/comfy_api_nodes/nodes_runway_text2img.py

```
@@ -0,0 +1,269 @@
+            endpoint=api_endpoint,
+            auth_kwargs=auth_kwargs,
+        ).execute()
+        
+        if response.status == TaskStatus.SUCCEEDED:
+            return True, response
+        elif response.status in {TaskStatus.FAILED, TaskStatus.CANCELLED}:
+            raise RuntimeError(f"Task failed with status: {response.status}")
+        return False, response
+    
+    start_time = datetime.now()
+    timeout_time = start_time + timedelta(seconds=timeout)
```

Ruff (0.12.2)

Local variable `timeout_time` is assigned to but never used

Remove assignment to unused variable `timeout_time`

(F841)